### DISCUSSION TITLE
Syntax error updating RLS through dashboard?

### DISCUSSION THREAD

**Discussion by pranavbadami** [supabase/supabase#2103] [Questions] [answered]
Hey team,

It seems like I'm getting a syntax error applying the following RLS policy via the dashboard, in this case on the `test` table:
```
EXISTS (
    SELECT
        1
    FROM users
    WHERE users.team = test.team 
    AND users.approved = TRUE 
    AND users.auth_users_uid = uid()
)
```

I already have these policies in my schema from before (entered in the old dashboard) and was trying to update policies on one of my tables. I now get the following error: 

`Error adding policy: syntax error at or near "("`

![image](https://user-images.githubusercontent.com/6540790/123532548-e04bde00-d6c2-11eb-8a5c-f24dcf693c90.png)

Any idea what could be the issue here? I'm using the same syntax I've used in the past for policies and I have currently active policies that use the same syntax. I also can't edit in the dashboard policy preview because it's read-only.

**Comment by d-mok** [comment]
can you show us the dashboard policy preview? what sql statement is supabase trying to execute?

**Reply by pranavbadami** [reply]
Hey, thanks for the reply! Here's the SQL that supabase shows me in the preview tab:

```
CREATE POLICY "Test" ON public.test FOR
SELECT
  USING (
    EXISTS (
      SELECT
        1
      FROM
        users
      WHERE
        users.team = test.team
        AND users.approved = TRUE
        AND users.auth_users_uid = uid()
    )
  );
```
Based on my intermediate level SQL, I don't see any obvious syntax errors here? Does it maybe have to do with the `EXISTS`?

**Comment by inian** [comment, marked as answer]
Hi @pranavbadami, this was an issue and has been fixed now. Thanks for the report!

**Reply by pranavbadami** [reply]
Awesome, thank you Inian!
